rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbService.java

```diff
@@ -16,4 +17,6 @@
 	 * @return A {@link CompletableFuture} that will hold the written Metadata.
 	 */
 	CompletableFuture<Metadata> write(Metadata meta);
+
+	public List<Metadata> getUnprocessedCpcPlusMetaData();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "public" is redundant in this context. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2333)